docs/guides/getting-started.md

```diff
@@ -34,57 +34,60 @@ We will use the [`@erc725/erc725.js`](../tools/erc725js/getting-started) NPM pac
 
 ```js title="Load web3"
 import Web3 from 'web3';
+import 'dotenv/config';
 
-const web3 = new Web3('https://rpc.l16.lukso.network');
+let privateKey = process.env.MY_PRIVATE_KEY;
+let address = process.env.MY_ADDRESS;
 
-const myDummyPassword = 'mypassword';
+const web3 = new Web3('https://rpc.testnet.lukso.network');
 
-// Here we try to load an already created key from the localstorage
-web3.eth.accounts.wallet.load(myDummyPassword);
+// Check if there is already a key, if none exists, we create a new one
+if (privateKey === '') {
+  let account = web3.eth.accounts.create();
 
-// If none exists we create a new key
-if (!web3.eth.accounts.wallet.length) {
-  web3.eth.accounts.wallet.create(1);
-  web3.eth.accounts.wallet.save(myDummyPassword);
```

Comment:
@biancabuzea200 Why did you remove this part?

Reply:
I tried with the old version today that uses wallet.save, wallet.load. I found that those are browser-only (https://web3js.readthedocs.io/en/v1.2.11/web3-eth-accounts.html), so the tutorial can only be run from the browser. With the .env setup, users have more flexibility over where they can run the tutorial from. Do we want to limit devs to only run in the browser?